services/user/CommonApi/common/packages/wit/host-common.wit

```diff
@@ -101,8 +101,15 @@ interface client {
     my-service-origin: func() -> string;
 }
 
+interface web {
+    use types.{error};
+
+    popup: func(url-path: string) -> result<_, error>;
```

Comment:
How would this work if we run a plugin in a non-browser context (like the CLI).

Reply:
It would not. That's why it's in an interface called, `web`. Also it's probably not the final interface, just a first pass to start playing with the functionality